sp800-63-3/sec5_xAL.md

```diff
@@ -209,17 +211,17 @@ All FALs require assertions to have a baseline of protections, including signatu
 
 RPs should use a back-channel [presentation mechanism](sp800-63c.html#sec6) where possible, as such mechanisms allow for greater privacy and security. Since the subscriber handles only an assertion reference and not the assertion itself, there is less chance of leakage of attributes or other sensitive information found in the assertion to the subscriber's browser or other programs. Since the assertion reference is presented by the RP directly to the IdP, the IdP can often take steps to identify and authenticate the RP during this step. Furthermore, since the assertion is fetched by the RP directly from the IdP over an authenticated protected channel, there are fewer opportunities for an attacker to inject an assertion into an RP.
 
-FAL 2 and higher require the assertion itself to be encrypted such that the intended RP is the only party that can decrypt it. This method not only improves the enforcement of audience restriction at RPs (since an unintended RP won't be able to decrypt an assertion), but also increases privacy protection by protecting the assertion message itself in addition to having it be passed along authenticated protected channels. RPs that allow front-channel presentation of assertions should require at least FAL 2 to protect the content of the assertion, since the assertion can be seen by the subsciber and handled by the subscriber's browser.
+FAL 2 and higher require the assertion itself to be encrypted such that the intended RP is the only party that can decrypt it. This method not only improves the enforcement of audience restriction at RPs (since an unintended RP won't be able to decrypt an assertion), but also increases privacy protection by protecting the assertion message itself in addition to having it be passed along authenticated protected channels. RPs that allow front-channel presentation of assertions should require at least FAL 2 to protect the content of the assertion, since the assertion can be seen by the subscriber and handled by the subscriber's browser.
 
-FAL 3 further requires that the subscriber prove posession of a key in addition to the ability to present an assertion or assertion reference. This method allows the RP to more strongly verify the subscriber's presenence through use of a second factor in addition to the assertion, where that second factor is tied to the assertion being presented.
+FAL 3 further requires that the subscriber prove possession of a key in addition to the ability to present an assertion or assertion reference. This method allows the RP to more strongly verify the binding of the subscriber's authenticator to the assertion.
```

Comment:
This new language makes it sound like the RP is verifying the subscriber's primary authenticator that was used at the IdP, which isn't the intent. It's meant to be additive.